webrender/src/device.rs

```diff
@@ -2049,9 +2049,8 @@ impl Device {
     }
 
     pub fn set_blend_mode_alpha(&self) {
-        //self.gl.blend_func(gl::SRC_ALPHA, gl::ONE_MINUS_SRC_ALPHA);
         self.gl.blend_func_separate(gl::SRC_ALPHA, gl::ONE_MINUS_SRC_ALPHA,
-                                     gl::ONE, gl::ONE);
+                                    gl::ONE, gl::ONE_MINUS_SRC_ALPHA);
```

Comment:
Will it not break the existing blended logic? Perhaps, we should have separate functions for the blend mode alpha and pre-multiplied alpha.